<div class="post_form col-lg-9 bg-white py-2 p-2">
  <div class="border rounded-left border-success p-3">
    <%= form_with model: @new_post, url: community_posts_path, local: true do |f| %>
      <% if @new_post.errors.any? %>
        <%= @new_post.errors.count %>件のエラーが発生しました
        <% @new_post.errors.full_messages.each do |message| %>
          <%= message %>
        <% end %>
      <% end %>
      <div class="row">
        <div class="form-group row col-lg-6 pr-0">
          <%= f.label :タイトル, class: "col-lg-4 col-form-label" %>
          <%= f.text_field :title, class: "col-lg-4 form-control" %>
        </div>
        <div class="col-lg-6">
          <%= f.label :画像, class: "text-right col-lg-2 col-form-label" %>
          <%= f.attachment_field :image , class: "col-lg-8" %>
        </div>
        <div class="form-group row col-lg-12 pr-3">
          <%= f.label :投稿内容を入力, class: "col-lg-2 col-form-label" %>
          <%= f.text_area :caption, class: "col-lg-4 form-control" %>
        </div>
        <%= f.hidden_field :community_id, value: @community.id %>
        <div class="col-lg-12">
          <%= f.submit "投稿内容を送信",class: "btn btn-outline-success btn-block" %>
        </div>
      </div>
    <% end %>
  </div>
</div>